### PR TITLE
Re-forming the PR from Demopans to better fit our repo structure.

### PR DESCRIPTION
**Pull request by TheMosquito**
I thought it would be too difficult to modify Dempans' PR so I decided to create this new PR instead.

Note that I added a credit to Demopans at the bottom of the WSL2.md file that contains the detailed instructions.

### CODE REVIEW

**Comment by Demopans** [comment]
It would probably be a good idea to add a disclaimer to readme.md for people installing this for WSL2

**Comment by TheMosquito** [comment]
@Demopans there is already a disclaimer in the mgmt-hub/README.md:

>The support for Windows WSL2 is also considered **experimental** because the WSL2 ubuntu environment is nonstandard. The WSL2 ubuntu is based on `init.d` instead of the `systemd` used in normal ubuntu distros. We require `systemd` support so we attempt to simulate systemd usiing [this script](https://gist.githubusercontent.com/djfdyuruiry/6720faa3f9fc59bfdf6284ee1f41f950/raw/952347f805045ba0e6ef7868b18f4a9a8dd2e47a/install-sg.sh). This technique may not be stable and reliable.

I had hoped that would make it clear that this is not a good idea for production use.

File: mgmt-hub/WSL2.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+## Simulating `systemd`
+
+The Open Horizon scripts use `systemd` to deploy daemon processes but the WSL2 ubuntu image uses `init.d` as PID 1 instead of `systemd`. This prevents the installation of our daemons. We found a work-around for this that simuulates a `systemd` environment in a WSL2 ubuntu environment.
+
+In a **WSL2 terminal shell**, run these commands to modify your WSL2 ubuntu to simuulate `systemd`:
+
+```bash
+sudo apt update
+
```
Comment (joewxboy) [inline mgmt-hub/WSL2.md:23]: sudo apt-get -y install wsl

**Comment by joewxboy** [comment]
Deprecated in favor of #139

**Pull request closed without merging**